templates/user/edit-profile.html

```diff
@@ -177,6 +200,78 @@
             });
         });
     </script>
+    <script src="{{ static('libs/clipboard/clipboard.js') }}"></script>
+    <script src="{{ static('libs/clipboard/tooltip.js') }}"></script>
+    <script type="text/javascript">
+        $(function () {
+            $('#generate-scratch-codes-button').click(function (event) {
+                event.preventDefault();
+                if (confirm("{{ _('Are you sure you want to generate or regenerate a new set of scratch codes?') }}\n"
```

Comment:
Batch these into one message like the other multi-line messages so translators have more context.